src/main/java/org/openelisglobal/samplebatchentry/controller/rest/SampleBatchEntrySetupRestController.java

```diff
@@ -0,0 +1,95 @@
+package org.openelisglobal.samplebatchentry.controller.rest;
+
+import java.lang.reflect.InvocationTargetException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.openelisglobal.common.formfields.FormFields;
+import org.openelisglobal.common.services.DisplayListService;
+import org.openelisglobal.common.services.DisplayListService.ListType;
+import org.openelisglobal.common.services.SampleOrderService;
+import org.openelisglobal.common.util.DateUtil;
+import org.openelisglobal.sample.controller.BaseSampleEntryController;
+import org.openelisglobal.sample.form.ProjectData;
+import org.openelisglobal.samplebatchentry.form.SampleBatchEntryForm;
+import org.openelisglobal.siteinformation.service.SiteInformationService;
+import org.openelisglobal.siteinformation.valueholder.SiteInformation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(value = "/rest/")
+public class SampleBatchEntrySetupRestController extends BaseSampleEntryController {
+
+    private static final String[] ALLOWED_FIELDS = new String[] {};
+
+    @Autowired
+    SiteInformationService siteInformationService;
+
+    @InitBinder
+    public void initBinder(WebDataBinder binder) {
+        binder.setAllowedFields(ALLOWED_FIELDS);
+    }
+
+    @RequestMapping(value = "ReactSampleBatchEntrySetup" ,produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
    @RequestMapping(value = "ReactSampleBatchEntrySetup" method = RequestMethod.GET,produces = MediaType.APPLICATION_JSON_VALUE)
```

Reply:
thank you let me make cahnnges @mherman22